code/game/objects/items/storage/secure.dm

```diff
@@ -187,3 +189,35 @@
 
 /obj/item/storage/secure/safe/HoS
 	name = "head of security's safe"
+
+/**
+ * This safe is meant to be damn robust. To break in, you're supposed to get creative, or use acid or an explosion.
+ *
+ * This makes the safe still possible to break in for someone who is prepared and capable enough, either through
+ * chemistry, botany or whatever else.
+ *
+ * The safe is also weak to explosions, so spending some early TC could allow an antag to blow it upen if they can
+ * get access to it.
+ */
+/obj/item/storage/secure/safe/caps_spare
+	name = "captain's spare ID safe"
+	desc = "In case of emergency, do not break glass. All Captains and Acting Captains are provided with codes to access this safe. \
+It is made out of the same material as the station's Black Box and is designed to resist all conventional weaponry. \
+There appears to be a small amount of surface corrosion. It doesn't look like it could withstand much of an explosion."
+	can_hack_open = FALSE
+	armor = list("melee" = 100, "bullet" = 100, "laser" = 100, "energy" = 100, "bomb" = 70, "bio" = 100, "rad" = 100, "fire" = 80, "acid" = 70);
```

Comment:
if the safe is going to be this strong, there should probably be more alternative methods for breaking in imo. Like thermite

Reply:
You can still break with thermite or explosion. The preferred method would be to kill acting captain and get spare that way. Numbers look strong we don't want to have people rushing fireaxe to break in.